### ISSUE TITLE
Drop down elements are not getting visual focus except the element in selected state ".NET Framework/ .NET Core" dropdown while navigating within the dropdown.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#340]
### Steps to Reproduce

Pre-Requisite: Change the system high contrast mode to black before launching the application.

1. Launch Xamarin Workbooks application. Which invokes "Create New Workbook" window.
2. Set the working platform to "Console" and navigate to ".NET Framework/.NET Core" dropdown and expand it.
3. Navigate within the dropdown.
### Expected Behavior
Visual keyboard focus should visible upon navigating to elements in the dropdown.
### Actual Behavior
No visual focus is observed for the drop elements in high contrast black mode.
Only element which is already in selected state is getting visual focus and is appearing in selected state. Rest of the elements are not getting any visual/keyboard focus.

Note: In High contrast white mode, dotted rectangular focus appears on dropdown elements (refer Hc white screenshot)
### User Impact
User will not whether it has got the focus or not. So low vision user will be confused.
### Screenshots
![5 hc black dropdown](https://user-images.githubusercontent.com/22956130/40722475-104ded54-643a-11e8-9035-6d5fcfc691e7.png)
![5 hc white dropdown focus](https://user-images.githubusercontent.com/22956130/40722476-1088bcf4-643a-11e8-98b3-03f2339dc47c.png)
![5 normal mode](https://user-images.githubusercontent.com/22956130/40722478-10c2a482-643a-11e8-8567-15dd65553e01.png)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.3.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
